python_frank_energie/frank_energie.py

```
@@ -1,16 +1,42 @@
 """FrankEnergie API implementation."""
-
-from __future__ import annotations
+# python_frank_energie/frank_energie.py
 
 import asyncio
-from datetime import date
-from typing import Any
+from datetime import date, timedelta
+from http import HTTPStatus
```

Ruff

`http.HTTPStatus` imported but unused

Remove unused import: `http.HTTPStatus`

(F401)